src/compas/rpc/proxy.py

```diff
@@ -37,28 +34,42 @@ class Proxy(object):
 
     Parameters
     ----------
-    package : :obj:`str`:, optional
+    package : str, optional
         The base package for the requested functionality.
-        Default is `None`, in which case a full path to function calls should be provided.
-    python : :obj:`str`:, optional
+        Default is None, in which case a full path to function calls should be provided.
+    python : str, optional
         The python executable that should be used to execute the code.
-        Default is ``'pythonw'``.
```

Comment:
Is this (and all other "Defaults to") removed because the default is visible directly in the method signature?

Reply:
yes